dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/SectionControl.mo

```diff
@@ -0,0 +1,37 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+model SectionControl
+import Modelica;
+parameter Real Ki "Integrator constant";
+parameter Real U0 "Input value at init (avoids potentially unnecessary control at startup)";
+parameter Real Y0 "Output value at init (avoids potentially unnecessary correction at startup)";
+  Modelica.Blocks.Interfaces.RealInput u(start=U0) annotation(
+    Placement(visible = true, transformation(extent = {{-140, -20}, {-100, 20}}, rotation = 0), iconTransformation(extent = {{-140, -20}, {-100, 20}}, rotation = 0)));
+  Modelica.Blocks.Tables.CombiTable1D combiTable1D(table = [0, 1; 1.0199, 1; 1.02, 1.02; 1.0499, 1.02; 1.05, 1.05; 1.0799, 1.05; 1.08, 1.08; 1.09, 1.08])  annotation(
```

Comment:
Instead of putting the table directly here, I suggest you use a txt file (you can have a look at GeneratorPVDiagramPQ for an example on how it is used).